example/gan/CGAN_mnist_R/CGAN_train.R

```diff
@@ -46,11 +47,11 @@ exec_D<- mx.simple.bind(symbol = D_sym, data=data_shape_D, digit=digit_shape_D,
 ### initialize parameters - To Do - personalise each layer
 initializer<- mx.init.Xavier(rnd_type = "gaussian", factor_type = "avg", magnitude = 3)
 
-arg_param_ini_G<- mx.init.create(initializer = initializer, shape.array = mx.symbol.infer.shape(G_sym, data=data_shape_G)$arg.shapes, ctx = mx.cpu())
-aux_param_ini_G<- mx.init.create(initializer = initializer, shape.array = mx.symbol.infer.shape(G_sym, data=data_shape_G)$aux.shapes, ctx = mx.cpu())
+arg_param_ini_G<- mx.init.create(initializer = initializer, shape.array = mx.symbol.infer.shape(G_sym, data=data_shape_G)$arg.shapes, ctx = devices)
```

Comment:
1. This line is too long
2. leave a space before `<-`

Please check the entire file with the above two comments.